composer.json

```diff
@@ -15,6 +15,7 @@
 		}
 	],
 	"require": {
+        "silverstripe/vendor-plugin": "^1.0",
```

Comment:
Does this module expose front-end resources? If not, I don't think this module is a requirement.

Also, are we expecting that modules are allowed to rely on framework supplying this as a requirement, or do we expect that every module maintainer has to add this line? I'd suggest that relying on framework to properly take care of the implications of using silverstripe-vendormodule would be an acceptable division of responsibility.

Reply:
Ah good point, removed.

Modules without exposed resources don't need the plugin.